ppb/events.py

```diff
@@ -271,6 +271,15 @@ class StopScene:
     scene: Scene = None
 
 
+@dataclass
```

Comment:
```suggestion
@dataclass(frozen=True)
```

Reply:
We actually can't freeze events, because of the feature allowing systems to modify events.

Reply:
OK, ok.